x-pack/plugins/fleet/server/collectors/agents_per_output.test.ts

```diff
@@ -19,33 +19,43 @@ jest.mock('../services', () => {
           { agents: 1, data_output_id: 'logstash1' },
           { agents: 1, monitoring_output_id: 'kafka1' },
           { agents: 1, data_output_id: 'elasticsearch2', monitoring_output_id: 'elasticsearch2' },
+          {
+            agents: 1,
+            data_output_id: 'es-containerhost',
+            monitoring_output_id: 'es-containerhost',
+          },
+        ],
+      }),
+    },
+    outputService: {
+      list: jest.fn().mockResolvedValue({
+        items: [
+          {
+            id: 'default-output',
+            is_default: true,
+            is_default_monitoring: true,
+            type: 'elasticsearch',
+          },
+          { id: 'logstash1', type: 'logstash' },
+          { id: 'kafka1', type: 'kafka' },
+          { id: 'elasticsearch2', type: 'elasticsearch' },
+          { id: 'es-containerhost', type: 'cloud' },
```

Comment:
nit: we do not have an output type cloud maybe worth using `elasticsearch` here to avoid some confusion